### ISSUE TITLE
Some error messages does not fits in popup window

### ISSUE THREAD

**Issue by JNechaevsky** [chocolate-doom/chocolate-doom#902]
Tested on SDL-2 latest dev build from Tue Mar 28 07:06:14 2017.

Some error messages does not fits in popup window. For example, trying to start:
chocolate-doom.exe -iwad any_pwad.wad

Gives following error message:
![Error](http://i.imgur.com/kCcdlvP.png)

This seems to be Windows 10 only bug, because on other OSes this is working just fine:

- Windows XP: http://i.imgur.com/H3OU4xn.png
- Windows 7: http://i.imgur.com/EpMC34R.png (+Aero theme)
- Windows 8.1: http://i.imgur.com/HOcXgTj.png
- Debian Linux / XFCE: http://i.imgur.com/qGjvaeb.png (only console output, no popup message)

**Comment by JNechaevsky** [comment]
Okay, I've made a more deep investigation and have a good news: this is a exclusively Windows 10 LTSB Russian edition bug. I have checked this on other Windows 10 editions:

- Windows 10 Pro (Russian): http://i.imgur.com/685PhSY.png
- Windows 10 LTSB (English): http://i.imgur.com/6UsCHBZ.png

Everything is okay with messages in both editions. Why this is happening in Russian LTSB - really no idea.
But knowing the fact that this is not Chocolate Doom, neither SDL2 bug is good enough. I haven't checked Home and Enterprise editions, because AFAIK Home is similar to Pro and Enterprise is similar to LTSB (actually, LTSB is Enterprise). No need to do this, since normally the customers will use only Home and Pro editions.

All I can suggest - no need to put any effort to fix this, this must be a Microsoft headache.
So let's just close this bug, considering this investigation is over.
